docs/samples/v1beta1/torchserve/model-archiver/README.md

```diff
@@ -0,0 +1,146 @@
+# Predict on a InferenceService using Torchserve
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/#install-kfserving).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+
+## Create PV and PVC
+
+This document uses amazonEBS PV
+
+### Create PV
+
+Edit volume id in pv.yaml file
+
+```bash
+kubectl apply -f pv.yaml
+```
+
+Expected Output
+
+```bash
+persistentvolume/model-pv-volume created
+```
+
+### Create PVC
+
+```bash
+kubectl apply -f pvc.yaml
+```
+
+Expected Output
+
+```bash
+persistentvolumeclaim/model-pv-claim created
+```
+
+### Create PV Pod
+
+```bash
+kubectl apply -f pvpod.yaml
+```
+
+Expected Output
+
+```bash
+pod/model-store-pod created
+```
+
+### Create properties.json file
+
+This file has model-name, version, model-file name, serialized-file name, extra-files, handlers, workers etc. of the models.
+
+```json
+[
+  {
+    "model-name": "mnist",
+    "version": "1.0",
+    "model-file": "",
+    "serialized-file": "mnist_cnn.pt",
+    "extra-files": "",
+    "handler": "mnist_handler.py",
+    "min-workers" : 1,
+    "max-workers": 3,
+    "batch-size": 1,
+    "max-batch-delay": 100,
+    "response-timeout": 120,
+    "requirements": ""
+  },
+  {
+    "model-name": "densenet_161",
+    "version": "1.0",
+    "model-file": "",
+    "serialized-file": "densenet161-8d451a50.pth",
+    "extra-files": "index_to_name.json",
+    "handler": "image_classifier",
+    "min-workers" : 1,
+    "max-workers": 3,
+    "batch-size": 1,
+    "max-batch-delay": 100,
+    "response-timeout": 120,
+    "requirements": ""
+  }
+]
+```
```

Comment:
This looks like the `TrainedModel` CR we have recently introduced, I'd suggest using `TrainedModel` and add necessary fields there so that [model agent](https://github.com/kubeflow/kfserving/tree/master/pkg/agent) can run the `torch-model-archiver` to package the model and generate the configuration files, this should eliminate the need of step to have user run the script to generate those.

```yaml
apiVersion: "serving.kubeflow.org/v1alpha1"
kind: "TrainedModel"
metadata:
  name: "mnist"
spec:
  inferenceService: torchserve
  model:
    framework: pytorch
    storageUri: s3://torchserve/mnist
    serializedFile: mnist_cnn.pt
    extraFiles: index_to_name.json
    handler: mnist_handler.py
    minWorker: 1
    maxWorker: 3
```

Reply:
@yuzisun We will do the Model Agent in the next round. Want to get the basic steps for Model archiver out first